src/ansys/dpf/core/operators/geo/normals_provider_nl.py

```diff
@@ -12,13 +12,13 @@
 
 
 class normals_provider_nl(Operator):
-    """Computes the normals on nodes/elements based on integration points
-    (more accurate for non-linear elements) on a skin mesh.
+    """Computes the normals on nodes/faces/elements based on integration
+    points (more accurate for non-linear elements) on a skin mesh.
```

Comment:
```suggestion
    """Computes the normals on nodes, faces, or elements based on integration
    points on a skin mesh.
    
    This method usings integration points because they are more accurate for non-linear elements.
```
We don't want to use "/" because it is unclear if it is repreesnts "and," "or," or "and/or" condition.
I also think this brief description for the class is too long. I hope what I moved to the second sentence is correct.